core/src/com/unciv/models/translations/Translations.kt

```diff
@@ -86,58 +86,58 @@ class Translations : LinkedHashMap<String, TranslationEntry>(){
             // which is super odd because everyone should support UTF-8
             languageTranslations = TranslationFileReader.read(Gdx.files.internal(translationFileName))
         } catch (ex: Exception) {
+            println("Exception reading translations for $language: ${ex.message}")
             return
         }
 
         // try to load the translations from the mods
         for (modFolder in Gdx.files.local("mods").list()) {
             val modTranslationFile = modFolder.child(translationFileName)
             if (modTranslationFile.exists()) {
-                val translationsForMod = Translations()
-                createTranslations(language, TranslationFileReader.read(modTranslationFile), translationsForMod)
-
-                modsWithTranslations[modFolder.name()] = translationsForMod
+                val translationsForMod = modsWithTranslations[modFolder.name()]
+                    ?: Translations().also {
+                        modsWithTranslations[modFolder.name()] = it
+                    }
+                try {
+                    translationsForMod.createTranslations(language, TranslationFileReader.read(modTranslationFile))
+                } catch (ex: Exception) {
+                    println("Exception reading translations for ${modFolder.name()} $language: ${ex.message}")
+                }
             }
         }
 
         createTranslations(language, languageTranslations)
 
         val translationFilesTime = System.currentTimeMillis() - translationStart
-        if(printOutput) println("Loading translation file for $language - " + translationFilesTime + "ms")
+        if (printOutput) println("Loading translation file for $language - " + translationFilesTime + "ms")
     }
 
-    private fun createTranslations(language: String,
-                                  languageTranslations: HashMap<String,String>,
-                                  targetTranslations: Translations = this) {
+    private fun createTranslations(language: String, languageTranslations: HashMap<String,String>) {
         for (translation in languageTranslations) {
             val hashKey = if (translation.key.contains('['))
                 translation.key.getPlaceholderText()
             else translation.key
-            if (!containsKey(hashKey))
-                targetTranslations[hashKey] = TranslationEntry(translation.key)
-
-            // why not in one line, Because there were actual crashes.
-            // I'm pretty sure I solved this already, but hey double-checking doesn't cost anything.
-            val entry = targetTranslations[hashKey]
-            if (entry != null) entry[language] = translation.value
+            val entry = this[hashKey] ?: TranslationEntry(translation.key).also { this[hashKey] = it }
```

Comment:
I'm sorry to harp in the same subject, but this is a good example.
For once, the .also allows for this to be shorter than the alternative.
But let's consider a hypothetical. Instead of an IDE, where mouseover solves my problems of typing, I happen to be viewing this from - I don't know - Github.
Then I come across this line. also{this...it}.
So wait, what's the 'it' in this case? What's the 'this'?
This is hardly the worst offender, but as an example, it serves